New "plot chart" visualization

nharada1 opened the pull request:
I created a new type of visualization called a "plot chart" that's basically a time chart but instead you can specify which variable to use for the x-axis (and there is no aggregation). Useful for charting processes that aren't uniform in real-time. 

Samy, if you aren't adding new features or have a specific set of features you'd like to retain, please feel free to deny this pull request, or email me (github username at gmail).

nharada1 committed:
Import source code

nharada1 committed:
Merge branch 'master' of https://git.heroku.com/mri-server

Conflicts:
	lib/apis/stats/time.js
	public/build/static/application.js
	public/src/views/visualizations/table.js
	public/src/views/visualizations/time.js

nharada1 committed:
Added a new 'plot chart' visualization

Plot chart is similar to a time chart but instead of aggregating
over time for the x-axis, it takes another field as the sampling
points and plots the remaining fields against that one. Good for
unevenly sampled data, non-time series data, etc.

SamyPesse commented:
I'm always happy to add new features :star2: , but I'm not really maintaining this project (I started working on a [new version](https://github.com/Reportr/dashboard/tree/version4) a few months ago, but I have too much work with other projects).

Is it ready to be merged?

If you're using Reportr, will you be interested in maintaining it ? I can add you as a collaborator of the organization so that you can easily move this project forward. Just let me know if you're interested :beers:

nharada1 commented:
It should be ready to be merged.

I really like Reportr (great job!), but I'm actually planning to fork it
for a project with slightly different aims. I'm a researcher at the
University of Michigan and we're developing a tool to let you monitor
training neural networks via web interface. I'll likely contribute some
changes over the summer, but our fork of the project will diverge as it
becomes more and more specific to our purposes.

Anyway, thanks so much for the work you've done so far! I'll be sure to
check out the new version once you get some more time to work on it. Maybe
once you start working on the new version I can help you too.

Best,

Nate

On Fri, May 15, 2015 at 4:19 PM, Samy Pessé notifications@github.com
wrote:

> I'm always happy to add new features [image: :star2:] , but I'm not
> really maintaining this project (I started working on a new version
> https://github.com/Reportr/dashboard/tree/version4 a few months ago,
> but I have too much work with other projects).
> 
> Is it ready to be merged?
> 
> If you're using Reportr, will you be interested in maintaining it ? I can
> add you as a collaborator of the organization so that you can easily move
> this project forward. Just let me know if you're interested [image:
> :beers:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Reportr/dashboard/pull/69#issuecomment-102516336.

nharada1 committed:
Deleting a report now correctly refreshes page to show other reports

Previously deleting a report would result in an error stating that no
reports existed. Now deleting a report reloads the default report.

nharada1 committed:
Merge branch 'master' of https://github.com/Reportr/dashboard into upstream-track

nharada1 commented:
Quick bug fix so that when deleting a report the page refreshes back to a valid report instead of saying there are no reports available.

nharada1 committed:
Check for existing reports when refreshing after deletion
